Laysa part 01

laysauchoa opened the pull request:
@marimeireles 

nao entendi essa parte:

> Uma última adição está no próprio ipywidgets: o novo widgetOutput pode exibir qualquer conteúdo que pode ser renderizado em um Jupyter notebook. Isso significa que qualquer coisa que você pode mostrar em um notebook, pode incluir em um widget usando apenas Python.

laysauchoa committed:
laysa/translation: eng to pt-br part 01

laysauchoa committed:
laysa/translation: eng to pt-br part 01

marimeireles commented on notebooks/01.00-overview.ipynb:
```diff
@@ -12,29 +12,29 @@
+    "# Visão geral\n",
     "\n",
-    "What you can accomplish with just Python has increased quite a bit in the last years as more sophisticated tools that plug in to the Jupyter widget ecosystem have been written.\n",
+    "O que você pode fazer usando apenas Python cresceu bastante nos últimos anos, à medida que ferramentas mais sofisticadas que se conectam ao ecossistema de widgets do Jupyter foram sendo escritas.\n",
```
aumentou bastante

laysauchoa replied:
blz, resolvido

marimeireles reviewed (changes requested):
Thank youuu

laysauchoa commented:
@marimeireles poderia olhar essa duvida que eu tive se vc souber? coloque na descrição no pr, obrigada :D

laysauchoa committed:
improve description

marimeireles commented:
>Uma última adição está no próprio ipywidgets: o novo widgetOutput pode exibir qualquer conteúdo que pode ser renderizado em um Jupyter notebook. Isso significa que qualquer coisa que você pode mostrar em um notebook, pode incluir em um widget usando apenas Python.

retirar

laysauchoa committed:
remove part of the text

this was not clear for us what does it mean,
pt/br:
"Uma última adição está no próprio ipywidgets: o novo widgetOutput pode exibir qualquer conteúdo que pode ser renderizado em um Jupyter notebook. Isso significa que qualquer coisa que você pode mostrar em um notebook, pode incluir em um widget usando apenas Python."

therefore, we have decided to remove those info.

laysauchoa commented:
@marimeireles pode revisar? ^^ removed.
